crates/router/src/connector/airwallex/transformers.rs

```diff
@@ -639,7 +641,7 @@ impl TryFrom<AirwallexWebhookEventType> for api_models::webhooks::IncomingWebhoo
                 Self::DisputeWon
             }
             AirwallexWebhookEventType::DisputeLost => Self::DisputeLost,
-            _ => Err(errors::ConnectorError::WebhookEventTypeNotFound)?,
+            _ => Self::EventNotSupported,
```

Comment:
I'd suggest not matching over `_` since new variants added to the enum would also end up being mapped to `EventNotSupported`, and the compiler won't be able to help you identify this problem.